service/certificate-generator/src/main/java/de/adorsys/psd2/sandbox/certificate/service/CertificateService.java

```diff
@@ -317,9 +318,9 @@ SubjectData generateSubjectData(CertificateRequest cerData) {
 
     X500NameBuilder builder = new X500NameBuilder(BCStyle.INSTANCE);
     builder.addRDN(BCStyle.O, cerData.getOrganizationName());
-
-    builder.addRDN(BCStyle.CN, "");
-
+    if(!StringUtils.isEmpty(cerData.getCommonName())) {
```

Comment:
isNotBlank?

Reply:
Yes, I know, but isNotBlank from apache.commons lib. Is it a good idea to add a dependency only because of one method?

Reply:
well, `org.springframework.util.StringUtils` is officially not recommended to use anywhere

https://docs.spring.io/spring-framework/docs/current/javadoc-api/index.html?org/springframework/util/StringUtils.html claims:
> Mainly for internal use within the framework; consider Apache's Commons Lang for a more comprehensive suite of String utilities.

So using it you put a potential bug in future, increasing technical debt